src/components/apps/launch/params/FolderInput.js

```diff
@@ -7,9 +7,14 @@ import React from "react";
 
 import DiskResourceSelector from "./DiskResourceSelector";
 import ResourceTypes from "components/models/ResourceTypes";
+import InfoTypes from "components/models/InfoTypes";
 
 export default function FolderInput(props) {
     return (
-        <DiskResourceSelector acceptedType={ResourceTypes.FOLDER} {...props} />
+        <DiskResourceSelector
+            acceptedType={ResourceTypes.FOLDER}
+            acceptedInfoTypes={[InfoTypes.HT_ANALYSIS_PATH_LIST]}
+            {...props}
```

Comment:
This might cover setting the `acceptedInfoTypes` correctly, I'm not 100% sure and it would need to be tested:

```
export default function FolderInput(props) {
    const {multiSelect, ...rest} = props;
    return (
        <DiskResourceSelector
            acceptedType={ResourceTypes.FOLDER}
            acceptedInfoTypes={!multiSelect ? [InfoTypes.HT_ANALYSIS_PATH_LIST] : null}
            multiSelect={multiSelect}
            {...rest}
```

Reply:
Looks good to me, although there is no need to keep `multiSelect` separate from the rest of the `props` since it still needs to be passed into the `DiskResourceSelector`.
```jsx
export default function FolderInput(props) {
    const { multiSelect } = props;
    return (
        <DiskResourceSelector
            acceptedType={ResourceTypes.FOLDER}
            acceptedInfoTypes={!multiSelect ? [InfoTypes.HT_ANALYSIS_PATH_LIST] : null}
            {...props}
```